assets/js/explosion-effect.js

```diff
@@ -0,0 +1,56 @@
+const randomBetween = (min, max) => {
+  return Math.floor(Math.random() * (max + 1)) + min
+}
+
+const createExplosionElement = (x, y) => {
+  let element = document.createElement("div")
```

Comment:
Use `const` instead `let` since you don't intend to change the variable reference.